_usage/migration.md

```diff
@@ -71,7 +71,7 @@ your database connections. Simply make your applications point to the new data
 source and launch your tests.
```

Comment:
previous paragraph: 'data structure' (multiple occurrences)  is the wrong terminology. Use 'schema' instead and change the header to : "Deploying the schema and migrate the data". Change "code" to "SQL code" or "SQL". Avoid wording like " Most of the code is expected to work..." because this may not be true for all customers. Remove the statement saying "moving the data itself is not expected to cause errors anymore" : this is what we're hoping, but there may be plenty issues left.

Reply:
Again, not my text, but I agree with the changes to "data structure", fixing the grammar of the headline and being more specific about "code".
I'll leave the weasel wording "Most of the code is expected to work" and "is not expected to cause errors anymore", as I think it is helpful. It shows where Amazon expects problems and where not, so that the user can get an idea what to expect.
I'll also leave the "colloquial" parts. I am used to PostgreSQL documentation, and that has no problem with entertaining the user, as long as it is not at the cost of accuracy, which I don't perceive here.

Reply:
I think this is not helpful. If you need to say that you think your code kinda works, that shines a negative light on it. It is software, so everyone understand that software comes with bugs. That doesn't mean we need to emphasize that. 

We should avoid entertainment, if only because whether something is entertaining is a matter of taste, so this is an area to be avoided in technical documentation.

Reply:
Ok, entertainment and weasel words removed.
You should ease up a bit!